docs/02-app/02-api-reference/02-file-conventions/layout.mdx

```diff
@@ -92,9 +92,9 @@ export default function ShopLayout({ children, params }) {
 
 ## Good to know
 
-### Layout's do not receive `searchParams`
+### Layouts do not receive `searchParams`
 
-Unlike [Pages](/docs/app/api-reference/file-conventions/page), Layout components **do not** receive the `searchParams` prop. This is because a shared layout is [not re-rendered during navigation](/docs/app/building-your-application/routing#partial-rendering) which could lead to stale `searchParams` between navigations.
+Unlike [Pages](/docs/app/api-reference/file-conventions/page), Layout components **do not** receive the `searchParams` prop. This is because a shared layout is [not re-rendered during navigation](/docs/app/building-your-application/routing/linking-and-navigating#partial-renderingg) which could lead to stale `searchParams` between navigations.
```

Comment:
```suggestion
Unlike [Pages](/docs/app/api-reference/file-conventions/page), Layout components **do not** receive the `searchParams` prop. This is because a shared layout is [not re-rendered during navigation](/docs/app/building-your-application/routing/linking-and-navigating#partial-rendering) which could lead to stale `searchParams` between navigations.
```